doc/articles/controls/AdaptiveTrigger.md

```diff
@@ -0,0 +1,148 @@
+# Using Adaptive Triggers With Uno Platform Applications
```

Comment:
I would suggest moving this to the `features` folder instead. Also, you need to add an entry pointing to this page from `toc.yml`

Reply:
I have moved the file from doc/articles/controls to doc/articles/features. Is there a way to add another commit to this pull request so that I can add an entry point to `toc.yml`? Or would it be better to delete this PR and start over?